controllers/authController.js

```diff
@@ -69,3 +69,34 @@ exports.register = async (req, res) => {
     })
   }
 }
+
+exports.resetPass = async (req, res) => {
+  try {
+    const userToSearchFor = { email: req.body.email, password: req.body.oldPass }
+    const newUserToSearchFor = { email: req.body.email, password: req.body.newPass }
+    const user = await req.db.User.findOne(userToSearchFor)
```

Comment:
You should search first only for email, if user not exists return a `404` and user not found, after that search if the user that you searched for has the same password with oldPassword.